### ISSUE TITLE
PHP on one line produce wrong indentation with one space too much.

### ISSUE THREAD

**Issue by louim** [FriendsOfPHP/PHP-CS-Fixer#1330]
Hello, I have a brunch of php file that are on a single line, for example: 

```
<?php  class SomeClass { const VERSION = '1.1.1'; const FOO = 'bar'; }
```
When formatted, I get:

```
<?php  class SomeClass
 {
     const VERSION = '1.1.1';
     const FOO = 'bar';
 }
```

It's a little bit hard to see, but the whole code after the PHP tag is indented one space too much.

I'm using the command like this: `php-cs-fixer fix Myfile.php`. 
Version is : `PHP CS Fixer version 1.10 by Fabien Potencier (8e21b4f)

**Comment by SpacePossum** [comment]
hmm this indeed (still) wrong on 1.10 but not on master it seems.

**Comment by keradus** [comment]
`T_OPEN_TAG` like to eat following spaces

**Comment by GrahamCampbell** [comment]
> T_OPEN_TAG like to eat following spaces

Yeh, it does. I really find that behaviour odd.

**Comment by SpacePossum** [comment]
This works on 1.11, 1.12 and master. I suggest closing this unless someone really wants to fix it on 1.10 as well.

There is another issue on 1.12 though with fixing the example.
Input:
```php
<?php  class SomeClass { const VERSION = '1.1.1'; const FOO = 'bar'; }
```

command; ` php php-cs-fixer fix test.php -vvv`

after iteration one;
```
   1) /home/possum/work/PHP-CS-Fixer/test.php (header_comment, braces, eof_ending)
```

re-run, iteration 2;
```
   1) /home/possum/work/PHP-CS-Fixer/test.php (blankline_after_open_tag)
```

I guess this covered by the other header fixer conflicts issues reported?

**Comment by keradus** [comment]
Thanks for checking this @SpacePossum !
1.10 is not supported.
